src/asm3/dbupdate.py

```diff
@@ -2591,6 +2604,34 @@ def user(tid: int, username: str, realname: str, password: str, superuser: bool)
     sql += lookup1("lksloglink", "LinkType", 4, _("Waiting List", l))
     sql += lookup1("lksloglink", "LinkType", 5, _("Movement", l))
     sql += lookup1("lksloglink", "LinkType", 6, _("Incident", l))
+    sql += medicaltype(1, _("Worm treatment", l), 0)
+    sql += medicaltype(2, _("Flea treatment", l), 0)
+    sql += medicaltype(3, _("Other parasite treatment", l), 0)
```

Comment:
can we make categories work? 

Parasite: Wormer
Parasite: Flea
Parasite: Other

Surgery: C-Section
Surgery: Other

hmmm